src/components/sbb-tag-group/sbb-tag-group.tsx

```diff
@@ -22,6 +22,11 @@ import { TagStateChange } from '../sbb-tag/sbb-tag.custom';
 export class SbbTagGroup implements ComponentInterface {
   @Element() private _element: HTMLElement;
 
+  /**
+   * This will be forwarded as aria-label to the inner list.
+   */
+  @Prop() public listAccessibilityLabel?: string;
```

Comment:
Prop is missing in storybook

Reply:
At this point I wonder if the `listAccessibilityLabel` property is needed at all, does it really allow the consumer to do something they couldn't do before?

Reply:
I as thinking about the same :) One advantage is that it would read that it is role list and not group. So a slight improvement. I also tested with role list on the tag group itself, but this doesn't work well. second advantage, you would directly land on the inner list instead of first the group and then the list.